src/pages/dar_application/DataUseAgreements.js

```diff
@@ -0,0 +1,105 @@
+import { a, div, fieldset, h, h3, label, li, ol, p, span} from 'react-hyperscript-helpers';
+import isNil from 'lodash/fp/isNil';
+import { Alert } from '../../components/Alert';
+import ReactTooltip from 'react-tooltip';
+
+export default function DataUseAgreements(props) {
+
+  const {
+    darCode,
+    showValidationMessages,
+    problemSavingRequest,
+    attestAndSave,
+    ConfirmationDialogComponent,
+    partialSave,
+    prevPage
+  } = props;
```

Comment:
Thank you, I love this pattern - makes it super clear what's required for the component.